.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,39 @@
+--
+name: 🐞 Bug Report
+about: Report a bug to help improve Sentry
+
+---
+
+<!--
+
+Do you want to ask a question? Are you looking for support? The Sentry message
+board is the best place for getting support: https://forum.sentry.io
+-->
+
+## Important Details
+
+* Sentry Version: XYZ
```

Comment:
Might also ask for SaaS vs Onpremise.

Reply:
Good idea. I added a checkbox list for the mode and version if the reporter is using on-premise.